telethon/client/telegrambaseclient.py

```diff
@@ -320,11 +320,20 @@ def __missing__(self, key):
         # Used on connection. Capture the variables in a lambda since
         # exporting clients need to create this InvokeWithLayerRequest.
         system = platform.uname()
+
+        if system.machine in ['x86_64','AMD64']:
+            default_device_model = 'PC 64bit'
+        elif system.machine in ['i386','i686','x86']:
+            default_device_model = 'PC 32bit'
+        else:
+            default_device_model = system.machine
+        get_system_version = lambda x : x[:x.index('-')] if '-' in x else x
+        default_system_version = get_system_version(system.release)
```

Comment:
Maybe it's better to just use `re.sub` and remove the lambda (which I presume is there to simply shorten the variable nMd).